telescope/selector.py

```diff
@@ -87,25 +91,70 @@ def _parse_file_contents(self, selector_file_contents):
     selector_input_json = json.loads(selector_file_contents)
     self.validate_selector_input(selector_input_json)
 
-    metrics = []
     if selector_input_json['metric'] == 'all':
```

Comment:
I think we should just support metrics as a list and get rid of the `all` meta-metric.